pycbc/distributions/sky_location.py

```diff
@@ -31,5 +35,56 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class Fisher():
```

Comment:
Does a more specific name make sense? It seems this applies only to sky locations and some some generic fisher matrix?

Reply:
@ahnitz this is actually the Fisher distribution on the sphere (special case of the von Mises-Fisher distribution in 3 dimensions, https://en.wikipedia.org/wiki/Von_Mises%E2%80%93Fisher_distribution) so I think the name is appropriate, but suggestions are welcome of course.

Reply:
I guess you would prefer FisherSky, to be consistent with UniformSky? Since the distribution expects the angular parameters to be called `ra` and `dec`, making it explicit that it is intended for sky locations seems appropriate.

Reply:
@Prasia-Pankunni please rename the class to FisherSky.

Reply:
Yes. the change is made.